lib/spoom/sorbet/sigils.rb

```diff
@@ -0,0 +1,39 @@
+# typed: true
+# frozen_string_literal: true
+
+module Spoom
+  module Sorbet
+    module Sigils
+      extend T::Sig
+
+      VALID_STRICTNESS = ["ignore", "false", "true", "strict", "strong"].freeze
```

Comment:
`__STDLIB_INTERNAL` is also a valid strictness even if only used for very specific purposes